src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let finalCost = 0;
+
+  for (let day = 0; day < days; day++) {
+    finalCost += 40;
+  }
+
+  if (days >= 7) {
+    finalCost -= 50;
```

Comment:
CL-1 
Also, what is 50? 
for example you can write variable  = discountMax or something different.

Reply:
and in the future it will be easier for you to reuse variables or change their value)